### DISCUSSION TITLE
引用问题

### DISCUSSION THREAD

**Discussion by LHT-Liu** [zotero-chinese/wiki#113] [求助 / Q&A] [answered]
<img width="520" alt="image" src="https://github.com/zotero-chinese/wiki/assets/64388319/5d3f7144-4af5-4589-bcbc-ca83879f5abb">
请问为什么我在word中引用，输入名字为什么不能直接自动显示出来，就是直接匹配出来，我还得自己下拉找半天

**Comment by northword** [comment, marked as answer]
首先显示文档中已经引用的，且与你关键词匹配的。
之后显示未引用的，匹配关键词的。

如果确认无法匹配关键词，可以向 Zotero 官方反馈（Zotero 官方论坛）

**Reply by LHT-Liu** [reply]
好的，感谢，我在研究一下
